operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -235,7 +336,8 @@ public void addCertToTrustStore(File certFile, String certAlias, File trustStore
     @Override
     public void addKeyAndCertToKeyStore(File keyFile, File certFile, String alias, File keyStoreFile, String keyStorePassword) throws IOException {
         // never called during the tests which use this MockCertManager
-        write(keyStoreFile, "key store");
+        //write(keyStoreFile, "key store");
+        Files.write(keyStoreFile.toPath(), ENTITY_CERT_STORE);
```

Comment:
Is this really needed? The commented out original code makes it confusing (and if nothing else, it should not be here I guess).

But also, your certificate is not a valid proper certificate for where this is used. So I wonder what advantages does this really bring since this is something what can potentially cause a lot of confusion later. While `key store` is obviously dummy certificate, an actual certificate might not be so clear as dummy value.